src/approaches/oracle_approach.py

```diff
@@ -93,96 +88,167 @@ def _get_options_by_names(env_name: str,
     return _get_from_env_by_names(env_name, names, "options")
 
 
-def _get_cover_gt_nsrts(options_are_typed: bool,
-                        include_robot_in_holding: bool = False,
-                        place_sampler_relative: bool = False) -> Set[NSRT]:
-    """Create ground truth NSRTs for CoverEnv.
+def _get_cover_gt_nsrts() -> Set[NSRT]:
```

Comment:
Thank you for refactoring this! It had been annoying to me how complicated it was getting. Looks really clean now!!